### ISSUE TITLE
showEmptyFilters not working

### ISSUE THREAD

**Issue by vanbroup** [CloudCannon/pagefind#62]
showEmptyFilters does not hide empty filter values with the following config:

```js
window.addEventListener('DOMContentLoaded', (event) => {
    new PagefindUI({ 
        element: "#search",  
        baseUrl: "/",
        showEmptyFilters: false
    });
});
```

Example:
https://pkic.org/blog/

**Comment by bglw** [comment]
Bah! I'll take a look 🙂 Apologies

**Comment by simonwiles** [comment]
Same issue.  Curiously I can't replicate itwith `npm run start` from the repo's `pagefind_ui/` folder, but building the code with `npm run build` produces an identical pagefind-ui javascript file to the one created by `npx pagefind`, and exhibits the same problem when used in my project.

**Comment by bglw** [comment]
Hello! This is fixed in the [Pagefind v0.9.0](https://github.com/CloudCannon/pagefind/releases/tag/v0.9.0) release 🎉
